src/pages/test-report/cat-adi-part2/components/vehicle-check/vehicle-check.ts

```diff
@@ -43,15 +44,22 @@ export class VehicleCheckComponent implements OnInit, OnDestroy {
 
   subscription: Subscription;
 
-  constructor(private store$: Store<StoreModel>) { }
+  faultCountProvider: FaultCountProvider;
+
+  constructor(
+    private store$: Store<StoreModel>,
+  ) {
+  }
 
   ngOnInit(): void {
 
+    this.faultCountProvider = new FaultCountProvider();
```

Comment:
any reason this is instantiated here?

Reply:
not sure...will investigate

Reply:
moved to provider and amended spec file appropreatly